vmdb/db/fixtures/ae_datastore/ManageIQ/Service/Provisioning/StateMachines/Methods.class/__methods__/dialog_parser.rb

```diff
@@ -0,0 +1,99 @@
+
+
+def vmdb_object_from_array_entry(entry)
+  model, id = entry.split("::")
+  $evm.vmdb(model, id.to_i) if model && id
+end
+
+def parent_task(task)
+  return task if task.miq_request_task.nil?
+  parent_task(task.miq_request_task)
+end
+
+def add_hash_value(sequence_id, option_key, value)
+  return if value.blank?
```

Comment:
@tinaafitz 
Is this check needed since we are already skipping nil values in line 41